qa-jcoin.planx-pla.net/portal/gitops.json

```diff
@@ -123,7 +123,7 @@
           "name": "Submit Data"
         },
         {
-          "link": "https://qa-jcoin.planx-pla.net/dashboard/Public/index.html",
```

Comment:
could you also delete the file at `qa-jcoin.planx-pla.net/dashboard/Public/index.html`

Reply:
@paulineribeyre  just deleted this file -- ready for merge?